data.sql

```diff
@@ -0,0 +1,8 @@
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES('Agumon', '2020-02-03', 0, true, 10.23);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES('Gabumon', '2018-11-15', 2, true, 8);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES('Pikachu', '2021-01-07', 1, false, 15.04);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES('Devimon', '2017-05-12', 5, true, 11);
```

Comment:
as seen in the queries the weight entry should be a float number and not a whole number as u represented above 👍🏾 please change it and put in the right entries